src/GmailClient.php

```
@@ -410,9 +416,9 @@ public function getAccountHealth(): array
      *
      * @param  bool  $lazy  Whether to return a lazy collection for memory-efficient iteration
      * @param  bool  $paginate  Whether to return a paginator for all results
-     * @return Collection<int, Label>|\Illuminate\Support\LazyCollection<int, Label>
+     * @return Collection<int, Label>|LazyCollection<int, Label>
      */
-    public function safeListLabels(bool $lazy = false, bool $paginate = false): Collection|\Illuminate\Support\LazyCollection
+    public function safeListLabels(bool $lazy = false, bool $paginate = false): Collection|LazyCollection
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$paginate'. (undefined)

(UnusedFormalParameter)